blog/_posts/2017-08-01-the-search-for-the-killer-app-of-unikernels.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "The search for the killer app of unikernels"
+authors: 
+- Per Buer
+author-urls: 
+- 'https://twitter.com/perbu'
+date: 2017-08-04 14:30:00
+tags: [unikernel]
+image: /images/steam_engine.jpg
+excerpt:  When a radically different technology comes along it usually takes time before we figure out how to apply it. 
+shorturl: 
+---
+{% include JB/setup %}
+
+When a radically different technology comes along it usually takes time before we figure out how to apply it. When we had steam engines running factories there was one engine in each factory with a giant driveshaft running through the whole factory. When the electric engine came along people started replacing the giant steam engine with a giant electric motor. It took time before people understood that they could deploy several small motors in different parts of the factory and connect electric cables rather than having a common driveshaft. It takes time to understand the technology and its applicability. 
+
+![Steam engine]({{BASE_PATH}}/images/steam_engine.jpg)
+
+The situration with unikernels is similar. We have this new thing and to some extent we're using it to replace some genereal purpose operating system workload. But we're still very much limited by how we think about operating systems and computers.
+
+Unikernels are radically different. Naturally the question of the killer app has come up on a number of occasions. As unikernels are quite different from the dominant operating systems of today it isn’t as easy to spot what the killer application for Unikernels will be. Here I’ll try to answer why this is hard.
```

Comment:
"what the killer application for Unikernels will be" -> "what their killer application will be" (no need to repeat).